Moose Development/Moose/Ops/ATIS.lua

```diff
@@ -302,8 +304,10 @@ ATIS = {
   subduration    =   nil,
   metric         =   nil,
   PmmHg          =   nil,
+  qnhonly        =   false,
```

Comment:
Not sure why these booleans default to `nil`, but I defaulted mine to `false` despite the convention. If there's a good reason why they should default to `nil`, let me know.

Reply:
Setting it to `false` as default is perfectly fine.